lowranceusr.cc

```diff
@@ -391,9 +907,25 @@ lowranceusr_parse_waypt(Waypoint* wpt_tmp)
     wpt_tmp->shortname = buff;
   }
 
-  if (global_opts.debug_level >= 1)
-    printf(MYNAME " parse_waypt: Waypt name = %s Lat = %f Lon = %f alt = %f\n",qPrintable(wpt_tmp->shortname), wpt_tmp->latitude,
-           wpt_tmp->longitude, wpt_tmp->altitude);
+  if (global_opts.debug_level > 1) {
+    if (global_opts.debug_level == 99) {
+      printf(" %16.16s %+15.10f %+15.10f",
```

Comment:
This is an awful lot of finicky formatting in debug code. Are we doing the debuggers's job for it? Could the finicky interleaving between string constants and debug values just be done with qDebug() ? Note http://doc.qt.io/qt-5/qdebug.html#formatting-options if you really want to display tenths of feet.
Maybe more readable to bust all this type of code into different functions and just pass the pointer to the wpt around.